iree/turbine/kernel/wave/decompose_dot_mma.py

```diff
@@ -55,7 +57,7 @@ def is_dot_mma(node: fx.Node) -> bool:
             lhs.index = copy(lhs_index)
             rhs.index = copy(rhs_index)
 
-            k_sym = get_custom(lhs).indexing_dims[1]
+            k_sym = get_custom(lhs).type.symbolic_shape[-1]
```

Comment:
how did this work before?

Reply:
It should be `-1` from the start but instead I wrote `1` so it breaks for non 2D shapes